### PR TITLE
add rainbow ink demo

### PR DESCRIPTION
**Pull request by conceptdev**
add another example handler and update the UI 

rainbow-800

### CODE REVIEW

File: inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt [hunk 1]
```diff
@@ -72,11 +96,15 @@ class MainActivity : AppCompatActivity() {
         var switch = view as Switch
         if (switch.isChecked) {
             inkView.dynamicPaintHandler = FancyPaintHandler()
+            // inkView.dynamicPaintHandler = HighlighterPaintHandler()
```
Comment (tggmbi) [inline inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt:99]: do we need this here? if its blog related, maybe add a comment to explain

**Review by tggmbi — APPROVED** [review]

File: inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt [hunk 2]
```diff
@@ -44,25 +46,47 @@ class MainActivity : AppCompatActivity() {
...
+    fun setRainbow(@Suppress("UNUSED_PARAMETER")view: View) {
+        inkView.dynamicPaintHandler = RainbowPaintHandler()
+        fancySwitch.isChecked = false
+    }
+    /**
+     * Required because some colors are set on the InkView, but
+     * rainbow ink is controlled via a paint handler (which is also
+     * how the pressure handler works, but it modifies color)
+     */
+    fun resetPaintHandler() {
```
Comment (CesarValiente) [inline inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt:82]: Even when `inkView` and `fancySwitch` are global variables that can be accessed along the class whenever we want, we maybe want to address the new reset-value inkView will get with a returned value so we don't hide the functionality of what happens when we call `resetPaintHandler`. For example something like:

```
fun setYellow(@Suppress("UNUSED_PARAMETER")view: View) {
        inkView.dynamicPaintHanlder = resetPaintHandler(fancySwitch.isChecked)
        inkView.color = Color.YELLOW
    }

fun resetPaintHandler (isFancySwitchChecked : Boolean) : FancyPaintHandler? =
       return if (isFancySwitchChecked) {
            FancyPaintHandler()
        } else {
            null
        }
}
```

**Review by CesarValiente — APPROVED** [review]
LGTM! Cool addition Craig!!

I just added a minor suggestion/comment.

File: inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt (hunk 1 above)
Comment (conceptdev) [reply inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt]: removed!

File: inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt (hunk 2 above)
Comment (conceptdev) [reply inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt:82]: @CesarValiente I like this approach - I propose we implement in another PR so I can ship this today 😉
Thanks for the feedback. The fix will be coming soon!

Comment (CesarValiente) [reply inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt:82]: Sure!! not a blocker at all! ;-)

**Comment by conceptdev** [comment]
Blog published!

https://devblogs.microsoft.com/surface-duo/android-pen-ink-control-preview/

**Pull request merged**